### DISCUSSION TITLE
What we did so far for tuning the classic content.

### DISCUSSION THREAD

**Discussion by 55Honey** [chromiecraft/chromiecraft#2438] [General]
**Tuning is a thing we did not want to do. We just need to do it.**

At a certain point, once a raid is no longer relevant for progression, we'll change the debuffs across raids accordingly.

Our plans are to test all encounters properly. There is no testing if an NPC/boss dies before it fired all of it's mechanics.
Without reduced damage, most encounters would be over before all mechanics were fired at all. For example i've seen Pyroguard Amberseer die in less than 5 Seconds, without debuffs. We can't fix a bug, If it never comes up!

The only difference between debuffing player damage and buffing boss HP is the scale of the numbers. 

If we would need to change every. Single. NPCs'. HP. And. dozens. other. stats. Manually. We would be busy for ages with that task.

Plus we could not at all work on fixing the actual NPC values, but we would need to work with arbitrary values and scale them back down to blizzlike to be able to apply them to AC. There is way too much effort in this and also way too many new sources for conversion mistakes.

So please bear with low dps numbers in progression. They're the same for everyone on ChromieCraft.

**Comment by Yehonal** [comment]
Thanks Honey for this.

I just wanted to add something to what you already said: 
a global tuning of the creature is "technically possible", that's what the autobalance module is intended for, but as mentioned on this topic,  we need to avoid changing the creatures as much as possible in order to test and fix them properly.
To summarize it up by quoting what Honey already said:

1. We can't massively change the creature stats for the reasons mentioned above

2. Tuning can't be optional, especially as regards the "end-game" content, since we need all the community to play the content in a way that is useful for the AC project.

3.  We will progressively remove the tuning once it's not relevant for the progression.

We will keep this approach until is **needed** for technical reasons.
